dev-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest
+httpx
+black
```

Comment:
feel free to add these to the main reqs file. My goal is to get CI to run tests before images get built but have been waiting for more tests to get written. Backend code needs to get to a point where it isn't changed as often for certain modules. (core modules)

Reply:
Hm.. I'm 50/50 on this one actually. Is this considered best practice? If so, then this is fine.

Reply:
Some folks put the dev requirements in `setup.py`, some make a second file, some use something like poetry to manage them.  I've found poetry to be more annoying than helpful in my day job.  I don't have any strong opinions on the approach so feel free to make a call.

Reply:
lol, I thought about checking out poetry. Thanks for saving me the hassle 😂 

We can leave it, thats fine then.

Reply:
We can just make two requirements text file, see many big projects with it

Also do we use httpx? I remember seeing Requests